sklearn/linear_model/tests/test_ransac.py

```diff
@@ -90,13 +89,16 @@ def test_ransac_max_trials():
                                        random_state=0)
     assert_raises(ValueError, ransac_estimator.fit, X, y)
 
-    ransac_estimator = RANSACRegressor(base_estimator, min_samples=2,
-                                       residual_threshold=5, max_trials=11,
-                                       random_state=0)
-    assert getattr(ransac_estimator, 'n_trials_', None) is None
-    ransac_estimator.fit(X, y)
-    assert_equal(ransac_estimator.n_trials_, 2)
-
+    # there is a 1e-9 chance it will take these many trials. No good reason
+    # 1e-2 isn't enough, can still happen
+    # 2 is the what ransac defines  as min_samples = X.shape[1] + 1
+    max_trials = _dynamic_max_trials(
+        len(X) - len(outliers), X.shape[0], 2, 1 - 1e-9)
```

Comment:
It would be easier to read without the need to refer to the docstring:

``` python
max_trials = _dynamic_max_trials(n_inliers=len(X) - len(outliers),                                                                                                                                                                           
                                 n_samples=len(X),                                                                                                                                                                                           
                                 min_samples=2,                                                                                                                                                                                              
                                 probability=1 - 1e-9)
```